src/features/donations/screens/TreeDonation.tsx

```diff
@@ -286,7 +337,6 @@ function TreeDonation({
                   onKeyPress={() => setOpenTaxDeductionModal(true)}
                   role="button"
                   tabIndex={0}
-                  data-testid="taxDeductionId"
```

Comment:
This is used by https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/106262e07ee0e156733bf9aacf9e8927c47bf122/__test__/TreeDonation.js#L18 (which is not yet an active test, but a demonstration).

Reply:
Can we add it again?

Reply:
Done